app/views/privacy_policy/index.html.slim

```diff
@@ -0,0 +1,33 @@
+- title t('titles.privacy_policy')
+
+h1.h3.mt0.mb1 = t('headings.privacy_policy.security')
+.mb4
+  p
+    | Lorem ipsum dolor sit amet Lorem ipsum dolor sit amet Lorem ipsum dolor
```

Comment:
Or, likely rename this to `index.en.html.slim` since line-for-line translation won't match up.